docs/reference/builder.md

```diff
@@ -121,6 +121,28 @@ registries.
 When you're done with your build, you're ready to look into [*Pushing a
 repository to its registry*](https://docs.docker.com/engine/tutorials/dockerrepos/#/contributing-to-docker-hub).
 
+
+## BuildKit
+
+Starting from version 18.09, Docker supports a new backend for executing your
```

Comment:
s/18.09/18.06/ ?

Reply:
I think it is too complicated to explain the need to enable experimental for `18.06`. If you want to just use BuildKit the recommendation is to upgrade to at least `18.09`.